rasa/KI-Campus_de/GreetBot_de/domain.yml

```diff
@@ -3,32 +3,42 @@ intents:
 - greet
 - goodbye
 - thank
-## added DFKI TODO enable when recommender is enabled
-#- utter_tip_formulation
+
+# # added by DFKI
+actions: 
+- utter_tip_formulation
+- utter_greet
+- utter_goodbye
+- utter_you_are_welcome
+- utter_have_a_nice_day
 
 responses:
-  utter_anything_else:
-  - text: Kann ich dir noch mit etwas anderem behilflich sein?
 
   utter_greet:
-   - text: Hey, wie kann ich dir helfen?
-   - text: Hey, was kann ich für dich tun?
-#  # modified by DFKI TODO replace above greetings with following text, when recommender is enabled
-#  - text: Hallo! 👋 Ich bin KIm, dein virtueller Assistent! Ich beantworte deine Fragen rund um den KI-Campus und helfe dir dabei, deinen Wunschkurs zu finden. Ich muss noch einiges dazulernen, deine Fragen werden mir dabei helfen. Frag mich also alles, was du zum KI-Campus wissen möchtest!
+#  - text: Hey, wie kann ich dir helfen?
+#  - text: Hey, was kann ich für dich tun?
+#  # modified by DFKI
+  - text: Hallo! 👋 Ich bin KIm, dein virtueller Assistent! Ich beantworte deine Fragen rund um den KI-Campus und helfe dir dabei, deinen Wunschkurs zu finden. Ich muss noch einiges dazulernen, deine Fragen werden mir dabei helfen. Frag mich also alles, was du zum KI-Campus wissen möchtest!
```

Comment:
Ich finde den neuen Text gut! Zur Zeit klingt es damit aber als ein reiner FAQ-Bot. Ich würde vorschlagen, den Teil "was du zum KI-Campus wissen möchtest" noch etwas umschreiben. Etwa in die Richtung "Was du zum Ki-Campus und deinem Lernerfolg wissen möchtest" (ohne ähnlich)

Reply:
Hallo! 👋 Ich bin KIm, dein virtueller Assistent! Ich beantworte deine Fragen rund um den KI-Campus**, deinen Lernerfolg** und helfe dir dabei, deinen Wunschkurs zu finden. Ich muss noch einiges dazulernen, deine Fragen werden mir dabei helfen. Frag mich also alles, was du zum KI-Campus wissen möchtest!

ODER Hallo! 👋 Ich bin KIm, dein virtueller Assistent! Ich beantworte deine Fragen rund um den KI-Campus und helfe dir dabei, deinen Wunschkurs zu finden. Ich muss noch einiges dazulernen, deine Fragen werden mir dabei helfen. Frag mich also alles, was du zum KI-Campus **und deinem Lernerfolg** wissen möchtest!

- Hast du eine Präferenz?

Reply:
Ich bin KIm, dein virtueller Assistent! Ich beantworte deine Fragen rund um den KI-Campus, zu deinen Kursen, und deinen Erfolgen. Außerdem helfe dir dabei, deinen Wunschkurs zu finden. Ich muss noch einiges dazulernen, deine Fragen werden mir dabei helfen. Frag mich also alles, was du wissen möchtest!

- Was hälst du davon?